clients/drcachesim/scheduler/scheduler.h

```diff
@@ -448,7 +448,7 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
          * specified by
          * #dynamorio::drmemtrace::scheduler_tmpl_t::scheduler_options_t::quantum_unit.
          */
-        uint64_t quantum_duration = 10 * 1000 * 1000;
+        uint64_t quantum_duration = 500 * 1000;
```

Comment:
Perhaps a quick note on why we think 500K is more appropriate (since we decided to switch from 1M to this, I assume there was a non-trivial reason).